iothub/service/tests/SerializationTests.cs

```diff
@@ -12,69 +15,249 @@ namespace Microsoft.Azure.Devices.Tests
     public class SerializationTests
     {
         [TestMethod]
-        public void Twin_JsonDateParse_Ok()
+        public void ClientTwin_JsonParse_Ok()
         {
-            const string expectedDeviceId = "test";
-            const string jsonString = @"
-{
- ""deviceId"": ""test"",
- ""etag"": ""AAAAAAAAAAM="",
- ""version"": 5,
- ""status"": ""enabled"",
- ""statusUpdateTime"": ""2018-06-29T21:17:08.7759733"",
- ""connectionState"": ""Connected"",
- ""lastActivityTime"": ""2018-06-29T21:17:08.7759733"",
-}";
-
-            ClientTwin clientTwin = JsonConvert.DeserializeObject<ClientTwin>(jsonString);
-            clientTwin.DeviceId.Should().Be(expectedDeviceId);
+            // arrange - act
+            var clientTwin = new ClientTwin("test")
+            {
+                ETag = new ETag("AAAAAAAAAAM="),
+                Version = 5,
+                Status = ClientStatus.Enabled,
+                StatusUpdatedOnUtc = new DateTimeOffset(2023, 1, 20, 8, 6, 32,
+                        new TimeSpan(1, 0, 0)),
```

Comment:
Style-wise, we either break all parameters to one line each, or none. There's plenty of line length left, so I think you can make this a one liner.
```suggestion
                StatusUpdatedOnUtc = new DateTimeOffset(2023, 1, 20, 8, 6, 32, new TimeSpan(1, 0, 0)),
```